engine/runtime-language-arrow/src/main/java/org/enso/interpreter/arrow/runtime/ArrowFixedSizeArrayBuilder.java

```diff
@@ -309,7 +248,13 @@ long getArraySize() {
 
   @ExportMessage
   boolean isArrayElementReadable(long index) {
-    return index >= 0 && index < size && !buffer.isNull((int) index);
+    return false;
+  }
+
+  @ExportMessage
+  Object readArrayElement(long index)
```

Comment:
This implementation (together with `isArrayElementReadable`) guarantees that ` builder.at 1` is going to yield an exception.